build.sbt

```diff
@@ -122,6 +122,15 @@ lazy val test = project
   .settings(allSettings)
   .settings(coverageExcludedPackages := "io\\.finch\\.test\\..*")
   .settings(
+    libraryDependencies ++= testDependencies
+  )
+  .dependsOn(core)
+
+lazy val `json-test` = project
```

Comment:
I think I'd prefer to use `json-test` for the directory name, `finch-json-test` for the module name, and `jsonTest` as the project name. This is entirely a matter of taste, but I think it makes sense to be idiomatic in each domain.

You can specify the directory explicitly (when it doesn't match the project name) with `project.in(file("json-test"))`.